app/src/main/java/org/wikipedia/feed/aggregated/AggregatedFeedContent.kt

```diff
@@ -15,4 +15,6 @@ class AggregatedFeedContent(
     @SerialName("mostread") val topRead: TopRead? = null,
     @SerialName("image") val potd: FeaturedImage? = null,
     val onthisday: List<OnThisDay.Event>? = null
-)
+) {
+    var randomOnThisDayEvent: OnThisDay.Event? = null
```

Comment:
Since this class is `@Serializable`, but we don't actually want this new field to be serialized, can we add `@Transient` to this field?

Reply:
Yes! Done.